integration-tests/jail/bin/tendermint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+tendermint node --p2p.persistent_peers "e04c6e7fcda7f3947179871053cec8de7b7a20cb@chain0:26656"
```

Comment:
can this be taken from environment variable instead of hardcoded?

Reply:
i'll move to env variable